Olympus.Stratis/functions/fn_addChatCommands.sqf

```diff
@@ -37,5 +37,10 @@
 }, "all"] call CBA_fnc_registerChatCommand;
 
 ["tac-repair", {
-    cursorObject setDamage 0;
+    if (cursorObject isKindOf "AllVehicles") then {
+        cursorObject setDamage 0;
+        ["ace_common_systemChatGlobal", "Repaired vehicle", ace_player] call CBA_fnc_targetEvent;
```

Comment:
Why? Player is already local.

Reply:
To be honest, I just took L21 as an example. Changed them L21, 42 & 44 to `CBA_fnc_localEvent`

Reply:
No, event is not necessary at all to show the message to the person that invoked it.